### PR TITLE
Code refactoring.

### PR DESCRIPTION
**Pull request by Anup1998**

### CODE REVIEW

File: report-auth/login-hook/demo.js [hunk 1]
```diff
@@ -8,17 +8,17 @@ visualize({
         loginFn: function (properties, request) {
             //use whatever you want to establish authentication 
             //be aware of same domain policy, 'request' works only with JRS instance
-            alert("Send custom login request. To 'https://mobiledemo.jaspersoft.com/jasperserver-pro/rest_v2/login'");
+            alert("Send custom login request. To 'https://infra-platforms-phase2-11723-rsarda.pfa.jaspersoft.com/jasperserver-pro/rest_v2/login'");
```
Comment (sprilukin) [inline report-auth/login-hook/demo.js:11]: is this ok to have urls like this?

File: inputControl-report/bind-with-report/demo.html [hunk 2]
```diff
@@ -3,6 +3,5 @@
 <div id="container">
     <div id="ic">
         <div><p>Loading...</p></div>
-    </div>
-    <div id="report"></div>
-</div>
\ No newline at end of file
+</div><div id="report"></div>
```
Comment (sprilukin) [inline inputControl-report/bind-with-report/demo.html:6]: looks like formatting has been lost here. maybe it's ok, just want you to know

File: dashboard-input/undo-redo/demo.html [hunk 3]
```diff
@@ -1,7 +1,6 @@
 <script type='text/javascript' src="https://mobiledemo.jaspersoft.com/jasperserver-pro/client/visualize.js"></script>
 
-<!-- JQuery - compiled and minified -->
-<script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+
```
Comment (sprilukin) [inline dashboard-input/undo-redo/demo.html:3]: so we are not using jquery anymore in examples right?
just want to confirm

File: report-export/json-export-d3/demo.html [hunk 4]
```diff
@@ -1,11 +1,9 @@
-<script src="https://code.jquery.com/jquery-2.1.0.js"></script>
+
 <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.4.8/d3.js"></script>
 <script src="https://rawgit.com/zigfred/dimple/master/dist/dimple.v2.1.2tmp.js"></script>
```
Comment (sprilukin) [inline report-export/json-export-d3/demo.html:3]: [rawgit.com](rawgit.com) reached end of it's life.
probably it's better to replace with direct github link: [https://raw.githubusercontent.com/zigfred/dimple/master/dist/dimple.v2.1.2tmp.js](https://raw.githubusercontent.com/zigfred/dimple/master/dist/dimple.v2.1.2tmp.js)
and even better it to fork this repo to tibco github in order to not lost it.

File: dashboard-input/undo-redo/demo.html (hunk 3 above)
Comment (Anup1998) [reply dashboard-input/undo-redo/demo.html:3]: We are providing jquery with fiddle itself. for example below URL will automatically include jquery.
https://jsfiddle.net/gh/get/jQuery/3.4.1/anup1998/JS-visualize/tree/master/report-export/json-export-d3/

File: report-export/json-export-d3/demo.html (hunk 4 above)
Comment (Anup1998) [reply report-export/json-export-d3/demo.html:3]: After replacing, fiddle shows "Cross-Origin Read Blocking (CORB) blocked cross-origin response https://raw.githubusercontent.com/zigfred/dimple/master/dist/dimple.v2.1.2tmp.js with MIME type text/plain. See https://www.chromestatus.com/feature/5629709824032768 for more details." this error in console.

File: report-auth/login-hook/demo.js (hunk 1 above)
Comment (Anup1998) [reply report-auth/login-hook/demo.js]: I've updated this.

File: inputControl-report/bind-with-report/demo.html (hunk 2 above)
Comment (Anup1998) [reply inputControl-report/bind-with-report/demo.html]: I've updated this.

File: report-export/json-export-d3/demo.html (hunk 4 above)
Comment (sprilukin) [reply report-export/json-export-d3/demo.html:3]: we can put that script to github pages

**Pull request merged**